routes/users.js

```diff
@@ -1,9 +1,94 @@
 const express = require('express');
+const { userWidgets, db } = require('../database/schema.js')
 const router = express.Router();
+const mongoose = require('mongoose')
+
 
 /* GET users listing. */
-router.get('/', (request, response, next) => {
-  response.send('respond with a resource');
+router.get('/', ( request, response, next ) => {
+  response.send( 'respond with a resource' );
+});
+
+/* GET widget */
+router.get('/widgets/:widgetId', ( request, response, next ) => {
+  const { widgetId } = request.params
+
+  userWidgets.find( {'widgets._id': widgetId},function(error, widgets){
+      return widgets
+  })
+  .then( data => console.log('finished') )
 });
 
+/* CREATE widgets */
+router.post('/:userId/widgets/create', (request, response, next ) => {
```

Comment:
We will follow a RESTful approach to API endpoints:

```
GET /resourceNamePlural - get all entries
POST /resourceNamePlural { data } - create a new entry
GET /resourceNamePlural/:id - get single entry
PUT /resourceNamePlural/:id { data } - update entry (technically, replace, but update is fine)
DELETE /resourceNamePlural/:id - delete entry
```

Please update endpoints to reflect this approach